src/mmds/src/token.rs

```diff
@@ -378,12 +378,23 @@ mod tests {
         // We allow a deviation of 20ms to account for the gap
         // between the two calls to `get_time_ms()`.
         let deviation = 20;
-        assert!(ttl >= MILLISECONDS_PER_SECOND - deviation && ttl <= MILLISECONDS_PER_SECOND);
+        assert!(
+            ttl >= MILLISECONDS_PER_SECOND - deviation && ttl <= MILLISECONDS_PER_SECOND,
+            "ttl={} not within +-{} of {}",
+            ttl,
+            deviation,
+            MILLISECONDS_PER_SECOND
+        );
 
         let time_now = get_time_ms(ClockType::Monotonic);
         let expiry = TokenAuthority::compute_expiry(0);
         let ttl = expiry - time_now;
-        assert!(ttl <= deviation);
+        assert!(
+            ttl <= deviation,
+            "ttl={} is greater than {}",
```

Comment:
nit: Use f-string here and above? Also seems like a perfect use of the `=` formatting:

```python
ttl = 100
limit = 50
print(f"{ttl=} is greater than {limit}")
ttl=100 is greater than 50
```

Reply:
Ah, this is a rust test, not a python one, and it seems `=` formatting is not a thing in rust :(

Reply:
Doh! Sorry, somehow I assumed this was Python 🤦